sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestHelper.scala

```diff
@@ -164,4 +170,305 @@ trait SQLQueryTestHelper extends Logging {
         (emptySchema, Seq(e.getClass.getName, e.getMessage))
     }
   }
+
+  /** A test case. */
```

Comment:
I assume these are just code move around, no actual changes?

Reply:
Yes, that's correct.